main/ZgatewayRF.ino

```diff
@@ -63,7 +63,11 @@ void setupRF() {
   ELECHOUSE_cc1101.setMHZ(CC1101_FREQUENCY);
   ELECHOUSE_cc1101.SetRx(CC1101_FREQUENCY);
 #  endif
+#  ifdef RF_DISABLE_TRANSMIT
+  mySwitch.disableTransmit();
+#  else
   mySwitch.enableTransmit(RF_EMITTER_GPIO);
+#  endif
   mySwitch.setRepeatTransmit(RF_EMITTER_REPEAT);
```

Comment:
Should we include the `mySwitch.setRepeatTransmit(RF_EMITTER_REPEAT);` into the `RF_DISABLE_TRANSMIT` scope ?

Reply:
It doesn't matter. The `disableTransmit` function sets the transmission PIN to `-1`. 
https://github.com/1technophile/rc-switch/blob/385a7e065f225c159bcd0d1b6278495423a1bd23/src/RCSwitch.cpp#L227-L229

The RCSwitch::send() function then checks if the transmission PIN is `-1` and returns if it is. 
https://github.com/1technophile/rc-switch/blob/385a7e065f225c159bcd0d1b6278495423a1bd23/src/RCSwitch.cpp#L554-L556

Setting the repeat value doesn't change the transmission PIN, so the transmit function stays disabled.